### PR TITLE
Table columns inheritance

### PR DESCRIPTION
**Pull request by vkorobkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,3 +271,75 @@ It is a quite simple:
...
+alex:
+  name: Alexander
+  role: ui
+  version: 1
+dimon:
+  name: Diman
+  role: qa
+  version: 1
+```
+`version: 1` was duplicated 3 times - for every single row. It might me convenient when you have 3 rows, but when
```
Comment (rodionovsasha) [inline README.md]: might be

File: src/main/java/com/github/vkorobkov/jfixtures/config/YamlConfig.java [hunk 2]
```diff
@@ -1,56 +1,96 @@
...
+
+    private <T> Optional<T> dig(String ... sections) {
+        List<String> splittedSections =
+            Arrays.stream(sections).
+            flatMap(s -> Arrays.stream(s.split(KEY_SEPARATOR))).
+            filter(s -> !s.isEmpty()).
+            collect(Collectors.toList());
+
+        if (splittedSections.isEmpty()) {
+            throw new IllegalArgumentException("Name can not be an empty");
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/config/YamlConfig.java]: Name can not be empty

File: src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java [hunk 3]
```diff
@@ -29,7 +36,7 @@
     }
 
     private boolean isFile(Path file) {
-        return !file.toFile().isDirectory();
+        return !Files.isDirectory(file);
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java:39]: Java 8's "Files.exists" should not be used
squid:S3725
The Files.exists method has noticeably poor performance in JDK 8, and can slow an application significantly when used to check files that don't actually exist.

Comment (vkorobkov) [reply src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java:39]: I don't get it: I don't use Files.exists in the place, at least, explicitly? I use `java.nio.file.Files#isDirectory` which does not have any warning it jre source/docs

File: README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 # JFixtures
 
 ## Preface
-Nowadays almost every project has acceptance/integration/other tests which use a dedicated test database with some test 
+Nowadays almost every project has acceptancpreviousration/other tests which use a dedicated test database with some test
```
Comment (rodionovsasha) [inline README.md]: acceptancpreviousration?

File: README.md [hunk 5]
```diff
@@ -271,3 +271,155 @@ It is a quite simple:
...
+  apply:
+    every_table_has_version:
+      # applies to every table from user_related_tables list and to comments and tickets
+      to: [*user_related_tables, comments, tickets]
+      concerns: has_version
+    # every_table_has_cr_date ...
+    # etc 
+```
+
+The items of `apply:to` could be a either a string or a regular expression.
```
Comment (rodionovsasha) [inline README.md:383]: `apply: to`

Comment (vkorobkov) [reply README.md:383]: Not sure, it's a section name where `:` is just a separator

Comment (rodionovsasha) [reply README.md:383]: :+1:

**Review by rodionovsasha — APPROVED** [review]

**Pull request merged**